docs/studios/unity/unity-sdk/in-editor-replay.md

```diff
@@ -4,10 +4,61 @@ sidebar_label: 'Bot Replay'
 
 # Replaying Bot Behaviors in the Unity Editor
 
-Bots running on Regression Games save all states observed and actions sent, which can be used in the editor
-for playback. This feature, combined with our validation framework, allows you to find and debug
-specific errors within your game.
+Bots running on Regression Games save all states observed, all actions sent, and all validations performed.  This data can be loaded in the editor for visualization and playback in the scene. This feature, combined with our validation framework, allows you to find and debug specific errors within your game.
 
-## The Bot Replay Pane
+## Downloading the Replay Data
 
-![timeline panel](timeline_panel.png)
\ No newline at end of file
+At the end of a Bot's lifecycle, the replay data for every game tick processed by that bot is saved into a single Zip file.  This file can be downloaded from the `History` section of the `Running Bots` tab in the Regression Games client.  Click the `Download Replay Data` button for the Bot instance you want to download.  This will download a file named `rg_bot_replay_data-<botInstanceId>.zip`.
+
+![replay data download](replay_data_download.png)
+
+## The Bot Replay Window
+
+### Opening the Bot Replay Window 
+With the Regression Games SDK installed in your Unity Editor Project, you can open the Regression Games `Bot Replay Window` from the `Regression Games` menu in the top bar.
+
+![rg top menu](rg_top_menu.png)
+
+### Loading your scene
+The Bot Replay Window will render the game replay state in the currently loaded editor scene.  For the best experience, make sure your gameplay scene is loaded in the editor so that all fixed scene assets will be visible and line up with the replay data.
+
+**Note:** _This replay process will dynamically create objects within the currently loaded scene.  Don't worry!  These objects will automatically be removed when running, saving, or building the scene.  You may occassionally be prompted to save the scene because it has been changed.  It is safe to save; the Bot replay objects will be removed from the scene before the save processes._
+
+![timeline feature](timeline.png)
+
+### Loading the replay data
+The first step in using the Bot Replay Window is loading the replay data Zip file.  Click the `Bot Replay Zip:` button in the top left of the window.  This will open a file chooser window.  Choose the `rg_bot_replay_data-<botInstanceId>.zip` file you downloaded from the Regression Game Client.
```

Comment:
```suggestion
The first step in using the Bot Replay Window is loading a replay data Zip file.  Click the `Bot Replay Zip:` button in the top left of the window.  This will open a file chooser window.  Choose the `rg_bot_replay_data-<botInstanceId>.zip` file you downloaded from the Regression Games Client.
```

Reply:
resolved